generated/nidaqmx/constants.py

```diff
@@ -624,7 +624,7 @@ class SensorPowerType(Enum):
 class ShuntCalSelect(Enum):
     A = 12513  #: Switch A.
     B = 12514  #: Switch B.
-    AAND_B = 12515  #: Switches A and B.
```

Comment:
Can you update CHANGELOG.md to mention this incompatibility. Don't worry too much about wordsmithing, we can take a final pass before release